app.py

```diff
@@ -0,0 +1,4 @@
+li = [1,2,3,4,5]
+def hi():
```

Comment:
defining a function hi( ) for the purpose to reuse it later in the code and improve its efficiency